npm/module.hlb

```diff
@@ -1,12 +1,10 @@
 export nodeModules
 
 fs install(fs src) {
-	image "node:alpine" with option {
-		resolve
-	}
+	image "node:alpine"
 	run "npm install" with option {
 		dir "/src"
-		mount src "/src"
-		mount fs { scratch; } "/src/node_modules" as nodeModules
+		mount src "/src" with readonly
+		mount scratch "/src/node_modules" as nodeModules
```

Comment:
I'm okay with these changes, but seems unintended.

Reply:
Yeah, quasi intended, I had them in my tree to commit, and figured I would just lump them here since they are trivial.